### PR TITLE
Added BackBee CMS

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Added BackBee CMS, based on Symfony components, Doctrine2 and Twig.

### CODE REVIEW

File: projects/en/backbee.html.twig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>
+BackBee is an open source CMS designed to revolutionize user experience.
+It uses “On Page Editing” architecture making content contribution intuitive and secure
+without sacrificing the platform’s performance.
+BackBee is also conceived to create "Out of the Box" websites that are highly responsive.
+BackBee is distributed under GNU terms of  the General Public License (or "GPL").
```
Comment (javiereguiluz) [inline projects/en/backbee.html.twig]: Minor error: there are two white spaces between these words -> `of  the`

File: projects/en/backbee.html.twig [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<p>
+BackBee is an open source CMS designed to revolutionize user experience.
+It uses “On Page Editing” architecture making content contribution intuitive and secure
```
Comment (javiereguiluz) [inline projects/en/backbee.html.twig]: Although "typographic quotes" are correct, in Symfony documentation we always use the plain straight quotes. So please, change `“On Page Editing”` by `"On Page Editing"` and `platform’s` by `platform's`. Thanks!

**Comment by mickaelandrieu** [comment]
@javiereguiluz fixed, thank you for review :)

**Comment by mickaelandrieu** [comment]
@javiereguiluz fixed, thank you for review :)

**Comment by javiereguiluz** [comment]
@mickaelandrieu thanks for adding this project to the list! I'm :+1: to add it because I know BackBee is being used in very large websites, its popularity is increasing and it uses a lot of Symfony components.

**Pull request merged**